### PR TITLE
New details added

### PR DESCRIPTION
**Pull request by lukasz-kniola**
Added restaurants and activities

How does it look?

### CODE REVIEW

File: lukasz-eastleigh.md [hunk 1]
```diff
@@ -1 +1,13 @@
 # Eastleigh, UK
+
+### Restaurants
+- Chimichanga
+- Frankie&Benny
+- Welcome Inn
```
Comment (r-brink) [inline lukasz-eastleigh.md]: @lukasz-kniola Nice, what food do they serve?

File: lukasz-eastleigh.md [hunk 2]
```diff
@@ -1 +1,13 @@
...
+
+### Activities
+- The Point (Theatre)
+- Swan Centre (Shopping)
+- Vue (Cinema)
```
Comment (r-brink) [inline lukasz-eastleigh.md:11]: @lukasz-kniola What movies do they show?

File: lukasz-eastleigh.md (hunk 1 above)
Comment (lukasz-kniola) [reply lukasz-eastleigh.md]: Descend pub grub

**Review by r-brink — APPROVED** [review]
Looks awesome! 👍 💯

**Pull request merged**